### PR TITLE
try increase timeout for create incident

### PR DESCRIPTION
**Pull request by erezh31**

### CODE REVIEW

File: Tests/test_integration.py [hunk 1]
```diff
@@ -137,12 +139,13 @@ def __create_incident_with_playbook(client, name, playbook_id):
     incidents = client.SearchIncidents(0, 50, 'id:' + inc_id)
 
     # poll up to 1 second
```
Comment (Bars92) [inline Tests/test_integration.py]: wrong comment

File: Tests/test_integration.py [hunk 2]
```diff
@@ -137,12 +139,13 @@ def __create_incident_with_playbook(client, name, playbook_id):
...
-    timeout = time.time() + 3
+    timeout = time.time() + 5
     while incidents['total'] != 1:
         incidents = client.SearchIncidents(0, 50, 'id:' + inc_id)
         if time.time() > timeout:
             print_error('failed to get incident with id:' + inc_id)
             return False
+        time.sleep(0.5)
```
Comment (Bars92) [inline Tests/test_integration.py]: now its really the same as fixed number of tries

**Comment by meirwah** [comment]
@erezh31 can you resolve conflicts on this PR?

**Comment by erezh31** [comment]
there is no need with this PR... changes in master are kind of the same

**Pull request closed without merging**